librelane/scripts/pyosys/ys_common.py

```diff
@@ -29,7 +29,7 @@
 
 
 def _Design_run_pass(self, *command):
-    ys.Pass.call__YOSYS_NAMESPACE_RTLIL_Design__std_vector_string_(self, list(command))
+    ys.Pass.call(self, list(command))
```

Comment:
This breaks Yosys versions below 0.59.1. LibreLane ships with Yosys 0.46. We're willing to consider this a backwards-compatible solution:

```suggestion
if hasattr(ys.Pass, "call"):
    ys.Pass.call(self, list(command))
else:
   ys.Pass.call__YOSYS_NAMESPACE_RTLIL_Design__std_vector_string_(self, list(command))
```